store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10357,4 +10357,9 @@ TODO: delete them permanently from here
   <globalConfigValue>TRUE</globalConfigValue>
   <desc>whether to enable CountObjectsRequest with userAccount/account type on Zimbra Admin Console</desc>
 </attr>
+
+<attr id="4098" name="zimbraFeatureAllowUsernameInPassword" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountInfo,accountCosDomainInherited" since="10.1.0">
```

Comment:
We also have same type of LC attribute in LC.java 
public static final KnownKey allow_username_within_password = KnownKey.newKey(true);

Why is separate LDAP attribute needed again?

Reply:
for Z8 & Z9 LDAP attrs cannot be updated in patch so released as LC and now this ldap attr can be released in major release for Z10. LC needs updating on all mailbox nodes but ldap attr can be updated globally.